docs/user_guide/index.md

```diff
@@ -0,0 +1,143 @@
+# Render Cinema 4D with AWS Deadline Cloud
+
+**User Guide & Getting Started**
+
+This guide will walk you through everything you need to know about using AWS Deadline Cloud with Cinema 4D - from installation to your first successful render.
+
+Transform your Cinema 4D rendering workflow with AWS Deadline Cloud - a fully managed service that scales your renders across hundreds of instances, freeing up your workstation for creative work.
+
+## The Cinema 4D Submitter
+
+![Cinema 4D Submitter Interface](images/submitter-dialog.png)
+
+*The Deadline Cloud submitter integrates directly into Cinema 4D's Extensions menu.*
+
+## Why Use Deadline Cloud for Cinema 4D?
+
+<div class="benefit-tabs">
```

Comment:
These docs look great!

Agree with @karthikbekalp 's suggestion to stick to plain Markdown for now. The custom tiles and carousel look great. In the near future though, I expect we'll have a standardized template for the user guides, or maybe build them all together into a single centralized site. It'll be easier to make that migration if there's less custom HTML and JS in the mix.